src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.interfaces.AreaCalculable;
+import core.basesyntax.interfaces.Drawable;
+
+public abstract class Figure implements Drawable, AreaCalculable {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.